packages/main/src/themes/TableCell.css

```diff
@@ -9,21 +9,21 @@
 }
 
 td {
-	padding: .5rem .25rem;
+	padding: inherit;
 	box-sizing: border-box;
 	word-break: break-word;
+	vertical-align: middle;
 }
 
-:host([first-in-row]) td,
 .ui5-table-popin-row td {
 	padding-left: 1rem;
 
 }
 
-:host([first-in-row]) td {
```

Comment:
With the removal of this, you will break the padding in the default (no selection) mode.
Instead of removal you would need to apply the class if the selection is None or something like that.

There is no padding now, used to have prior this change:

<img width="212" alt="Screenshot 2021-02-19 at 8 11 59 PM" src="https://user-images.githubusercontent.com/15702139/108544151-baff2b80-72ee-11eb-87c8-503ccf3aa27c.png">

<img width="162" alt="Screenshot 2021-02-19 at 8 14 02 PM" src="https://user-images.githubusercontent.com/15702139/108544366-04e81180-72ef-11eb-9540-261354364995.png">

Reply:
Done